README.md

```diff
@@ -1,85 +1,232 @@
-=======================================
-Djangoflow Chat
-=======================================
+# Djangoflow Chat
 
 Opinionated Django Chat
 
+## Requirements
 
+> **_NOTE:_**  Generated content, as with human generated content, may contain errors or oversights. Please review before merging!
 
+Before integrating `django-df-chat` into your Django project, ensure that the following prerequisites are met:
 
+1. [Python](https://www.python.org/): This project requires Python 3.8 or newer.
+2. [Django](https://www.djangoproject.com/): Django version 3.2 or newer is required.
+3. [DjangoFlow](https://github.com/djangoflow): Django Flow, our base project with integrated CI/CD, is a recommended starting point for incorporating `django-df-chat`.
```

Comment:
This package can work without base djangoflow project, so I think we can remove this from dependencies